src/rlx_util.erl

```diff
@@ -39,19 +41,63 @@
 
 -include("rlx_log.hrl").
 
+relx_sasl_vsn() ->
+    application:load(sasl),
+    case application:get_key(sasl, vsn) of
+        {ok, SaslVsn} ->
+            %% TODO: change to actual version of sasl with systools changes
+            parse_vsn(SaslVsn) > {{infinity, infinity, infinity}, [], []};
```

Comment:
is there ever a case where this doesn't just return `true`?

Reply:
I guess that was the thing to replace in the TODO

Reply:
Hm? it won't because the atom is greater than a number. So this way it is always false until we insert the version that has the changes in place of infinity.

Reply:
yeah that, I just swapped the boolean.